spec/features/home_page_spec.rb

```diff
@@ -0,0 +1,29 @@
+# home page
+require 'pg'
+
+feature 'Viewing home page' do
+  scenario 'Introductory title' do
+    visit('/')
+    expect(page).to have_content 'Welcome to Chitter'
+    expect(page).to have_content 'Peepdeck'
+  end
+end
+
+# feature 'Viewing home page' do
+#   scenario 'Shows peepdeck' do
+#     visit('/')
+#     expect(page).to have_content 'My first peep!'
+#   end
+# end
+
+feature 'Viewing home page' do
+  scenario 'Shows peepdeck' do
```

Comment:
You can place this scenario inside of the first feature, as it is testing the same feature